index.html

```diff
@@ -46,20 +47,177 @@ <h1 class="headline-title">I'm Uzair.<br>Glad to see you!</h1>
                     </li>
                     <li class="social-links">
                         <a href="#">
-                            <img src="resources/icon/Vector.svg" alt="">
+                            <img src="resources/icon/Vector.svg" alt="vector">
                         </a>
                     </li>
                     <li class="social-links">
                         <a href="#">
-                            <img src="resources/icon/VectorM.svg" alt="">
+                            <img src="resources/icon/VectorM.svg" alt="vectorm">
                         </a>
                     </li>
                 </ul>
             </div>
                 </div>
             </article>
         </section>
-    </main>
-    
+        <!-- work section -->
+        <section class="work-section">
+            <div class="work-card">
+                <div class="work-preview">
+                    <img class="project-img" src="resources/images/Snapshoot Portfolio 1.png" alt="Project 1">
+                </div>
```

Comment:
- [ ] [OPTIONAL]  I have noticed that in all of your project cards you are using `div` tags around a single element `img` tag and I would suggest you to remove them.  Unless you need to include the `div` for some structural/organisational reason you're likely better off just styling the element tag itself or you can just use class name or id names for styling that component. The purpose of `divs` is to allow you to group elements together. Wrapping individual elements in `divs` for the purpose of styling is largely unnecessary.